### PR TITLE
Development

### PR DESCRIPTION
**Pull request by andreimorosan29**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    linear-gradient(
+        #8a8a8a, 
+        #707070 50%,#5e5e5e 50%, 
+        #787878);
+    padding: 8px;
+    border:none;
+    border-right: 1.5px solid rgb(53, 53, 53);
+    text-shadow: 0px -2px 0px #4a4a4a, 0px 2px 0px #505050;
+    box-shadow: 0px 5px 5px 0px black;
+    font-size: 12px;
```
Comment (heyapricot) [inline css/styles.css:57]: Property font-size is overwritten

Comment (heyapricot) [reply css/styles.css:57]: L38

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    background-image:
+    linear-gradient(
+        #8a8a8a, 
+        #707070 50%,#5e5e5e 50%, 
+        #787878);
+    padding: 8px;
+    border:none;
+    border-right: 1.5px solid rgb(53, 53, 53);
+    text-shadow: 0px -2px 0px #4a4a4a, 0px 2px 0px #505050;
+    box-shadow: 0px 5px 5px 0px black;
```
Comment (heyapricot) [inline css/styles.css]: You can omit the `px` when using a value of 0

File: index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="main-content">
+        <div class="topbar">
+            <button type="button" class="topbar-button"><i class="fab fa-apple"></i></button>
```
Comment (heyapricot) [inline index.html:17]: The buttons don't do anything when they are `hovered`. In the original, the background gets darker and the cursor changes when the button hovers.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            <button type="button" class="topbar-button">iPad</button>
+            <button type="button" class="topbar-button">iPhone</button>
+            <button type="button" class="topbar-button">iTunes</button>
+            <button type="button" class="topbar-button">Support</button>
+            <form action="#" class="topbar-search">
+                <input type="search" placeholder="Search">
+            </form>
+        </div>
+        <div class="header">
+            <h1>What will your verse be?</h1>
```
Comment (heyapricot) [inline index.html:29]: This header is different from the rest of the text in the original page. Just changing the tag to `h1` doesn't seem enough

File: index.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="main-content">
```
Comment (heyapricot) [inline index.html:15]: Try to match the width of the original page. In your version this div is narrower

File: index.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <div class="header">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the
+                world’s story.</h2>
+            <h3>Watch the “Your Verse” film and explore the stories</h3>
+        </div>
+    </div>
+    <div class="footer-grid">
+        <div class="footer-element">
```
Comment (heyapricot) [inline index.html:36]: Why did you decide to go for a gradient and not an image?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <div class="footer-element">
+            <div><strong style="font-size: 22px;">Supplier Responsibility</strong></div>
+            <br>
+            <div>The Latest result of our worldwide effors
+            </div>
+        </div>
+    </div>
+    <div class="footer">
+            Shop the Apple Online Store (1-800-MY-APPLE), visit an Apple Retail Store, or find a reseller.Site MapHot NewsRSS FeedsMedia InfoEnvironmentJob OpportunitiesContact UsChoose your country or region Copyright © 2014 Apple Inc. All rights reserved. Terms of UsePrivacy Policy
```
Comment (heyapricot) [inline index.html]: This seems just like a copy paste. The original has links and a layout for them. Try to be as close to the original as you can

**Review by heyapricot — CHANGES_REQUESTED** [review]
# Good first attempt. Needs a bit more of work
Hello! 👋🏼 You are on the right path, but the project looks different enough from the original page that I have to request for you to invest more time in this project. Please address the comments, and after that submit a new code review request. Thanks! 

Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 6 above)
Comment (htuerker) [reply index.html:36]: That's in the project name :) We follow through the guidelines. Also, we wanted to show our learnings about it.

**Comment by rvvergara** [comment]
Hi again Burhan and Andrei,

Once again you did awesome here. So once again, as well, I only have minor stuff to say:

1. Just like in your TNW clone, please populate your README with these additional information (as I also mentioned in your TNW repo):
 - short description of the project. What feature do you wish to showcase here? This gives a viewer a short but meaningful introduction on what to expect with your project
- Your names and links to your respective repositories (refer to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet on how to use markdown for including links to your names)
- A link to your project's live demo. Normally we use https://raw.githack.com to do this (this tool generates a link for your project so that anyone can see your final product live)

2. Your html code (index.html) passed all the validations. Awesome!

3. Great work on the linear gradients! And may I add, great choice for the jumbotron image (I don't know if that's the same image in the original but still I like it!). 

I have nothing more to add. You did a really really awesome job. You guys rocked this project. So yeah, just edit your README and the world deserves to see your live demo and know who you guys are! :) I'm gonna approve this one but you gotta really make sure your README's all filled up. ayt?

:+1:

You may merge this into your Master (upon editing your README).

**Pull request merged**